lib/openers/sumatra-opener.js

```diff
@@ -5,15 +5,9 @@ import Opener from '../opener'
 
 export default class SumatraOpener extends Opener {
   open (filePath, texPath, lineNumber, callback) {
-    const sumatraPath = `"${atom.config.get('latex.sumatraPath')}"`
-    const args = [
-      '-forward-search',
-      `"${texPath}"`,
-      lineNumber,
-      `"${filePath}"`
-    ]
+    const command = `"${atom.config.get('latex.sumatraPath')}" -forward-search "${texPath}" "${lineNumber}" "${filePath}"`
```

Comment:
I'd really prefer to keep the `sumatraPath` and `args` variables as it makes it a little bit easier to reason about how the command is constructed. The actual `command` variable itself could then perhaps be pieced together as
```js
const command = `"${sumatraPath}" ${args.join(' ')}`
```